client/src/pages/Review/Review.tsx

```diff
@@ -0,0 +1,113 @@
+    console.log("star 2 is checked")
+}else {
+    console.log("star2 is not checked")
+}
+
+// then apply gold styles to star-1
+
+
+// if input id star 3 is checked
+
+// then apply gold styles to star-2 and star-1
+
+return(
+    <div>
+        <h1>Review for Business Name Placeholder</h1>
+        <form className="stars">
+            <label>
+                <h3>Rate this restaurant overall:</h3>
+            <input type="radio" name="star-rating" aria-label="One Star" id="1-star" value="1"/>
+            <img src={emptyStar} alt="One Star Rating" />
+            </label>
+
+             <label>
+            <input type="radio"  name="star-rating" aria-label="Two Star" id="2-star" value="2"/>
+            <img src={emptyStar} alt="Two Star Rating" />
+            </label>
+            
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Three Star" id="3-star" value="3"/>
+            <img src={emptyStar} alt="Three Star Rating" />
+            </label>
+
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Four Star" id="4-star" value="4"/>
+            <img src={emptyStar} alt="Four Star Rating" />
+            </label>
+
+            <label>
+            <input type="radio"  name="star-rating" aria-label="Five Star" id="5-star" value="5"/>
+            <img src={emptyStar} alt="Five Star Rating" />
+            </label>
+
+        <div className="good-for">
+            <h3>What type of food is this restaurant good for? </h3>
+            <p>Please only select if the restaurant offers more than 3 options of that type.</p>
+            <p>Choose all that apply.</p>
+            <label>
+            <input type="checkbox"  name="vegan" aria-label="vegan" id="vegan" value="1"/>
+            Vegan
+            </label>
+
+            <label>
+            <input type="checkbox"  name="vegetarian" aria-label="vegetarian" id="vegetarian" value="1"/>
+            Vegetarian
+            </label>
+
+            <label>
+            <input type="checkbox"  name="pescatarian" aria-label="pescatarian" id="pescatarian" value="1"/>
+            Pescatarian
+            </label>
+
+            <label>
+            <input type="checkbox"  name="gluten-free" aria-label="gluten-free" id="gluten-free" value="1"/>
+            Gluten Free
+            </label>
+
+            <label>
+            <input type="checkbox"  name="dairy-free" aria-label="dairy-free" id="dairy-free" value="1"/>
+            Dairy Free
+            </label>
+
+            <label>
+            <input type="checkbox"  name="nut-free" aria-label="nut-free" id="nut-free" value="1"/>
+            Nut Free
+            </label>
+            
+        </div>
+
+
+        <label>
+            <h3>How was your experience?</h3>
+            <textarea className="paragraph"  name="review" aria-label="review" id="review"/>
+
+            </label>
+
+        <br />
+        <br />
+
+<button type="submit">Submit</button>
```

Comment:
@Darolo13 yes I need help with this. I looked at how you used Button in your login page but I keep getting typescript error: Type '{ content: string; name: string; }' is not assignable to type 'IntrinsicAttributes'.
[1]   Property 'content' does not exist on type 'IntrinsicAttributes'

Reply:
This is what I tried: ` <Button content="Submit" name="test"  />`

Reply:
Without testing myself, I think it's because you might not be able to pass a string directly as a prop like that, you can try to save the string as a variable first, then pass it through, so it would like this:
```
let submit = 'Submit';
let test = 'test';
... 
<Button content={submit} name={test} />
```

Reply:
Thanks, not sure why it's working now but it is! <ButtonStyles type='submit'>Submit</ButtonStyles>